api_tests/applications/views/test_application_detail.py

```diff
@@ -1,35 +1,49 @@
+import pytest
 import mock
 
-from nose.tools import *  # flake8: noqa
-
 from website.models import ApiOAuth2Application, User
 from website.util import api_v2_url
-
 from tests.base import ApiTestCase
+from api_tests.utils import assert_dict_contains_subset
+from tests.json_api_test_app import JSONAPITestApp
 from osf_tests.factories import ApiOAuth2ApplicationFactory, AuthUserFactory
 
 
 def _get_application_detail_route(app):
-    path = "applications/{}/".format(app.client_id)
+    path = 'applications/{}/'.format(app.client_id)
     return api_v2_url(path, base_route='/')
 
 
 def _get_application_list_url():
-    path = "applications/"
+    path = 'applications/'
     return api_v2_url(path, base_route='/')
 
+@pytest.mark.django_db
+class TestApplicationDetail:
+
+    @pytest.fixture()
+    def app(self):
```

Comment:
This will be used in almost every test, so let's make this a "global" fixture (i.e. move it to `conftest.py`).